BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -654,7 +654,7 @@ be used to avoid the risk of successful extraction of the signing key.
 ### Implementation Details
 {: #detail-neg}
 
-Clients SHOULD include TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 as the first proposal to any server, unless they have prior knowledge that the server cannot respond to a TLS 1.2 client_hello message.
+Clients SHOULD include TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 as the first proposal to any server.
```

Comment:
This would be better as a single paragraph with the next sentence, or even with the one following.

Reply:
Agreed, see https://github.com/yaronf/I-D/pull/391/commits/68f5d7515a1936756eb4ecf4321cdb6b3f2a4993